recipe/meta.yaml

```diff
@@ -20,16 +20,17 @@ build:
 requirements:
   build:
     - python
+    - pip
```

Comment:
Is this needed beyond py 2.7.9 and 3.4? I'm uncertain how Continuum have packaged ```pip```, but to my knowledge, it is now part of the stdlib, right?

Ping @msarahan - do you have any insight?